### ISSUE TITLE
Critical erron on website when using Caldera Forms Stripe: 1.4.11

### ISSUE THREAD

**Issue by AngelMatilla** [mecachisenros/caldera-forms-civicrm#185]
**Describe the bug**
After successfully submitting a form, being redirected to the stripe payment gateway, performing the payment correclty, the redirection back to the wordpress website shows the following display: "There has been a critical error on your website"

**To Reproduce**
1. Install wordpress WordPress Version: 5.5.1 (PHP Version: 7.3.16, MySQL Version: 5.7.24)
2. Install and activate the following plugins in Wordpress:  
Caldera Forms: 1.9.2
CiviCRM: 5.29.0
Caldera Forms Stripe: 1.4.11 
CF CiviCRM: 1.0.5
3. Create a simple form with a stripe processor and publish it
4. Fill in the form and pay (also works in test mode)
5. When the stripe gateway redirects back to the website, a critical error appears and the form is not logged (no entry).  

If the CF CiviCRM plugin is deactivated, this error does not happen. 

**Expected behavior**
The payment returns to a correct working website and the submitted form is logged. 

**Desktop (please complete the following information):**
 - OS: Linux XPS-13-9360 5.4.0-51-generic #56-Ubuntu SMP Mon Oct 5 14:28:49 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
 - Browser Chromium
 - Version Version 86.0.4240.75 (Official Build) snap (64-bit)

**Comment by dev-allinappli** [comment]
exactly the same issue I wanted to post. Have you found a solution?

After many tests, Caldera Form is incompatible with Enable Jquery Migrate and CF-civicrm :-/

**Comment by cdhassell** [comment]
> exactly the same issue I wanted to post. Have you found a solution?
> 
> After many tests, Caldera Form is incompatible with Enable Jquery Migrate and CF-civicrm :-/

Is this related to the original post? Why do you say CF is incompatible with Enable Jquery Migrate?

**Comment by kcristiano** [comment]
Can you test just with Caldera Forms and Stripe 1.4.11 ?  I have not had this configuration work consistently.   

I have also not had issues with Caldera and enable jQuery migrate plugin so I'd be interested in the issues @allinappli has had.

**Comment by dev-allinappli** [comment]
> > exactly the same issue I wanted to post. Have you found a solution?
> > After many tests, Caldera Form is incompatible with Enable Jquery Migrate and CF-civicrm :-/
> 
> Is this related to the original post? Why do you say CF is incompatible with Enable Jquery Migrate?

Yes it is related to the original and I was adding, which is a bit off-topic, that CF-Stripe is also not compatible with jQuery Migrate Helper that is needed to have caldera forms functioning on the lastest WP version from 5.6 even without cf-civicrm enabled.   Which means I'm twice concerned. 
[EDIT]: not so sure anymore about enable jQuery migrate, but it is minor.

**Comment by dev-allinappli** [comment]
I confirm that payment works with ONLY :
caldera forms + cf-stripe 1.4.11 + caldera license manager OR caldera forms + cf-stripe 1.4.11 + caldera license manager + cf-civicrm (all latest version of plugins) (I've disabled all other WP plugins and reactivating each one in a row still worked)

but once civicrm (5.35.1 or 5.37.2) is enabled it doesn't work anymore and I'm getting the same error message.

**Comment by dev-allinappli** [comment]
> Can you test just with Caldera Forms and Stripe 1.4.11 ? I have not had this configuration work consistently.
> 
> I have also not had issues with Caldera and enable jQuery migrate plugin so I'd be interested in the issues @allinappli has had.

I have to say that my priority is CiviCRM and cf-civicrm is the cornerstone of our websites. Some don't use need payments but for those who use Stripe, it's a big issue. 
@kcristiano: I gave a try with only caldera forms + cf-stripe 1.4.11 + caldera license manager + cf-civicrm + enabl.e jQuery and it worked. Maybe I mistook yesterday. But as said my topic is really civicrm

**Comment by kcristiano** [comment]
@allinappli Thanks for the details.   Are you using `cf-civicrm` from the wp.org repo or GitHub?  if GitHub, what branch/commit are you on?

**Comment by dev-allinappli** [comment]
I'm using the one (1.05) from the wp.org repo. I saw that there is a more recent version (1.11) on git hub, I'll give a try now.

**Comment by dev-allinappli** [comment]
I'm getting the same issue with the version 1.11 (that still shows up as 1.05), when civicrm is enabled

**Comment by dev-allinappli** [comment]
@kcristiano Should I try an other version? Is there a difference between the 1.05 on WP and GitHub?

**Comment by kcristiano** [comment]
I am using the dev branch - https://github.com/mecachisenros/cf-civicrm/tree/dev   you can see the detailed commits here: https://github.com/mecachisenros/cf-civicrm/commits/dev

yes the version has not been bumped, but as long as you are on the dev branch you are on the latest code.

I expect there will be a release once all testing is complete.  Both WP 5.5+ and Caldera 1.9.4 introduced a good deal of 'breaking changes'  which I am sure has delayed these updates.   Personally, I have also had issues with cf-stripe 1.4.11 so I am unsure where the issue precisely is.

**Comment by dev-allinappli** [comment]
@kcristiano in one word: thanks! it works with dev version and civicrm 5.35.1. you saved my week ;-)
